README.md

```diff
@@ -20,6 +20,22 @@ To test:
 
 You should receive a `wwan0` interface, with an IP, and a default route.
 
+# Complete example for Ubuntu 20.04
+
+```
+mkdir ~/tmp/
+cd ~/tmp/
+sudo apt install build-essential python3-pyroute2 python3-configargparse git
+git clone https://github.com/xmm7360/xmm7360-pci.git
+cd xmm7360-pci
+make && make load
+sudo python3 rpc/open_xdatachannel.py --apn gprs.swisscom.ch
```

Comment:
Should probably use the generic `your.apn.here`

Reply:
done thanks for the imput